hw/opentitan/ot_spi_device.c

```diff
@@ -43,10 +43,10 @@
 #include "hw/riscv/ibex_irq.h"
 #include "trace.h"
 
-#define PARAM_SRAM_DEPTH         1024u
-#define PARAM_SRAM_OFFSET        4096u
-#define PARAM_SRAM_EGRESS_DEPTH  832u
-#define PARAM_SRAM_INGRESS_DEPTH 104u
+#define PARAM_SRAM_OFFSET        0x1000u
+#define PARAM_SRAM_DEPTH         (0x1000u / sizeof(uint32_t))
+#define PARAM_SRAM_EGRESS_DEPTH  (0xe00u / sizeof(uint32_t))
+#define PARAM_SRAM_INGRESS_DEPTH (0x200u / sizeof(uint32_t))
```

Comment:
Likewise, from the [docs](https://opentitan.org/earlgrey_1.0.0/book/hw/ip/spi_device/doc/registers.html#ingress_buffer), this should maybe be:

```suggestion
#define PARAM_SRAM_INGRESS_DEPTH (0x1c0u / sizeof(uint32_t))
```
or just `112u`.